jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsAsBooleanTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common.util;
```

Comment:
Your class's package name is wrong:
```suggestion
package org.eclipse.jkube.kit.common;
```

Reply:
added changes.